core/src/util.rs

```diff
@@ -218,7 +218,6 @@ macro_rules! debug {
 }
 
 // Only needed in tests
-#[cfg(test)]
```

Comment:
`cfg(test)` doesn't get set frfom outside the crate, and we need this impl in the wasm crate tests too now.